src/OSPSuite.Core/Extensions/DimensionsListExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using OSPSuite.Core.Domain.UnitSystem;
+
+namespace OSPSuite.Core.Extensions
+{
+   public static class DimensionsListExtensions
+   {
+      public static IDimension DimensionForUnit(this List<IDimension> dimensionList, string unitName)
```

Comment:
I dont get why we have two overloads. Cannot you just work with one using Ienumrable. Specifically because the first entry is a query with a tolist.

Reply:
good point - I was thinking the other way around (had already the function taking a List as an argument and wanted to avoid the ".ToList()" call on the "IEnumerable").  I have removed the redundant overload